_includes/templates/download.html

```diff
@@ -251,21 +249,16 @@ <h2 style="text-align: center">{{page.build_reproduction}}</h2>
 <script type="text/javascript">
 var os = 'windows64';
 if (navigator.userAgent.indexOf('Mac') != -1) var os = 'mac'
-else if (navigator.userAgent.indexOf('Linux') != -1) {
-	var os='linux32';
```

Comment:
I guess we don't need this check for ARM, as the detection assumes Linux is always x86?

Reply:
I think we assume that only x86 can run a browser :grimacing:

Reply:
Created an issue: https://github.com/bitcoin-core/bitcoincore.org/issues/696